src/test/java/org/openrewrite/java/testing/mockito/ReplacePowerMockitoIntegrationTest.java

```diff
@@ -15,26 +15,28 @@
  */
 package org.openrewrite.java.testing.mockito;
 
-import static org.openrewrite.java.Assertions.java;
-
 import org.junit.jupiter.api.Test;
 import org.openrewrite.InMemoryExecutionContext;
 import org.openrewrite.config.Environment;
 import org.openrewrite.java.JavaParser;
 import org.openrewrite.test.RecipeSpec;
 import org.openrewrite.test.RewriteTest;
 
+import static org.openrewrite.java.Assertions.java;
+
 class ReplacePowerMockitoIntegrationTest implements RewriteTest {
     @Override
     public void defaults(RecipeSpec spec) {
         spec
           .parser(JavaParser.fromJavaVersion()
+            .logCompilationWarningsAndErrors(true)
             .classpathFromResources(new InMemoryExecutionContext(),
               "mockito-core-3.12.4",
               "junit-jupiter-api-5.9.2",
               "junit-4.13.2",
-              "powermock-core-1.7.4",
-              "powermock-api-mockito-1.7.4",
+              "powermock-core-1.6.5",
+              "powermock-api-mockito-1.6.5",
+              "powermock-api-support-1.6.5",
               "testng-7.7.1"))
```

Comment:
```suggestion
              "testng-7.7.1"))
          // TODO Resolve the missing types in the replacement templates rather than ignore the errors here
          .typeValidationOptions(TypeValidation.builder()
            .identifiers(false)
            .methodInvocations(false)
            .build())
```